example/session/src/main/java/org/apache/iotdb/SessionExample.java

```diff
@@ -352,6 +353,24 @@ private static void query() throws IoTDBConnectionException, StatementExecutionE
     dataSet.closeOperationHandle();
   }
 
+  private static void rawDataQuery() throws IoTDBConnectionException, StatementExecutionException {
+    List<String> paths = new ArrayList<>();
+    paths.add("root.sg1.d1.s1");
+    paths.add("root.sg1.d1.s2");
+    paths.add("root.sg1.d1.s3");
+    long startTime = 10L;
+    long endTime = 200L;
+
+    SessionDataSet dataSet;
+    dataSet = session.executeRawDataQuery(paths, startTime, endTime);
+    System.out.println(dataSet.getColumnNames());
+    dataSet.setFetchSize(1024); // default is 10000
```

Comment:
It's better not comment on the end of the line

Reply:
removed. Thank you!